### DISCUSSION TITLE
Start CandlestickSeries higher on Y-Axis

### DISCUSSION THREAD

**Discussion by somguyth** [reactivemarkets/react-financial-charts#536] [General]
Is there a way to get `CandlestickSeries` to start higher-up on the Y-Axis, instead of exactly at the bottom of it?

By default, the price candles block out the volume candles, [seen here](https://ibb.co/V3XJ9zk), whenever the price & volume are displayed on the same graph.

In TradingView, their price chart starts about 1/8th the way up the screen, so as to give room to the volume bars.

**Comment by markmcdowell** [comment]
yeah you can but you need to specify a `yExtentsCalculator` for the `Chart` and then you can pad the data.

See https://github.com/reactivemarkets/react-financial-charts/blob/308d3a605fddd3acd162f27bf492f8e67f703b48/packages/stories/src/features/updating/BasicLineSeries.tsx#L55 as an example
